torch/csrc/distributed/c10d/SymmetricMemory.cpp

```diff
@@ -269,6 +274,10 @@ TORCH_LIBRARY_FRAGMENT(symm_mem, m) {
       "stream_write_value32_(Tensor(a!) input, int offset, int val) -> Tensor(a!)");
   m.def(
       "memset32_(Tensor(a!) input, int offset, int val, int count) -> Tensor(a!)");
+
+  m.def("nvshmem_broadcast(Tensor(a!) input, str group_name) -> Tensor(a!)");
+  m.def(
+      "nvshmem_reduce_scatter_out(Tensor input, str group_name, Tensor(a!) out) -> Tensor(a!)");
```

Comment:
Why do these two APIs have the `nvshmem_` prefix? Do we plan to unify `nvshmem_reduce_scatter_out` with `reduce_scatter_out`?

Reply:
Correct. I expect we can remove the prefix by the release.